services/galley/src/Galley/API/Teams/Features.hs

```diff
@@ -545,7 +569,7 @@ setFileSharingInternal ::
     Member (Input Opts) r
   ) =>
   FeatureSetter 'TeamFeatureFileSharing r
-setFileSharingInternal = Tagged $ \tid status -> do
+setFileSharingInternal = Tagged $ \tid status ttl -> do
```

Comment:
Maybe it's not very important, but something to consider, when the TTL expires, and the feature is disabled again, this will not trigger a push event. We should check if this can cause any problems, when clients do not get this info via push event right away.

Reply:
For this specific feature or any feature?